server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/lisk/wallets/liskbnb/Lskwallet.java

```diff
@@ -0,0 +1,171 @@
+package com.generalbytes.batm.server.extensions.extra.lisk.wallets.liskbnb;
+
+import com.generalbytes.batm.server.extensions.Currencies;
+import com.generalbytes.batm.server.extensions.Currencies;
+import com.generalbytes.batm.server.extensions.IWallet;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import si.mazi.rescu.HttpStatusIOException;
+import si.mazi.rescu.RestProxyFactory;
+
+import org.apache.commons.codec.binary.Hex;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.UnsupportedEncodingException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+import java.util.Random; 
+
+public class Lskwallet implements IWallet {
```

Comment:
Why use Binance as a hot wallet? I'd suggest implementing `IExchange` interface. This will greatly enhance flexibility of LSK support. You can use it to refill your hot wallet or (with the correct exchange strategy) directly in place of your hot wallet.

Reply:
Thank you for your suggestion.
I am planning to add lsk wallet's option using lisk core 1.0, which was released a few days ago, and implement IExchange interface for binance in the coming month. But for now, I would like to use binance as an alternative.